distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -995,6 +951,82 @@ requiredAttributes=["/\=","/admin\={http://schemas.xmlsoap.org/ws/2005/05/identi
 whiteListContexts=["/services/SecurityTokenService","/services/internal/metrics","/services/saml","/proxy","/services/csw"]
 ----
 
+==== Configuring Global System Properties
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through the system properties on startup. After changing any of these properties you will need to restart the system. If you change the 'hostname' property you will also need to configure the certs as described in Configuring {branding} with New Certificates.
+The system by default uses both http and https so both httpsPort and httpPort need to be specified. The protocol and port properties are the defaults the system should use in places where either http or https could be valid.
+----
+org.codice.ddf.system.protocol=https://        #one of http:// or https://
```

Comment:
Do we have a ticket to combine this with other sections that discuss those properties?

This should also be converted to a table (should be done as part of this ticket if we don't have a separate one).